src/languages/en.ts

```diff
@@ -707,7 +707,7 @@ const translations = {
     reportTypingIndicator: {
         isTyping: 'is typing...',
         areTyping: 'are typing...',
-        multipleUsers: 'Multiple users',
+        multipleUsers: 'Multiple members',
```

Comment:
Since the variable implies that the string will be "multiple users" and not "multiple members," could we update the variable as well? Looks like we only use it here: https://github.com/Expensify/App/blob/45a19bf059d66f1c34c1da6a3ccd82545c679997/src/pages/home/report/ReportTypingIndicator.tsx#L51

Reply:
Did that work?

Reply:
I think so, but we also need to update the spanish translation file I think with the new key. Do we need a new translation too?

https://github.com/Expensify/App/blob/main/src/languages/es.ts#L704

Reply:
OK, I updated there, too. I'm not sure where else I'm needed, but please let me know!